docs/source/contributing/tasks.md

```diff
@@ -84,35 +90,26 @@ See the subsections below for more detailed instructions.
    ```
 
    This script will resolve dependencies and write them to the respective `.frozen.yml`
-   files. You will need `docker` installed to run this script.
+   files.
 
-3. After the freeze script finishes, a number of files will have been created.
-   Commit the following subset of files to git:
+3. After the freeze script finishes, a number of files will have been **modified**.
+   They roughly follow this pattern:
+
+   ```
+   repo2docker/buildpacks/conda/environment*
+   ```
 
-    ```
-    repo2docker/buildpacks/conda/environment.yml
-    repo2docker/buildpacks/conda/environment.frozen.yml
-    repo2docker/buildpacks/conda/environment.py-2.7.yml
-    repo2docker/buildpacks/conda/environment.py-2.7.frozen.yml
-    repo2docker/buildpacks/conda/environment.py-3.5.frozen.yml
-    repo2docker/buildpacks/conda/environment.py-3.6.frozen.yml
-    ```
+   You should **commit all modified files** to git.
 
-5. Make a pull request; see details below.
+4. Make a Pull Request to the `jupyterhub/repo2docker` repository, with a description
+   of what versions were bumped / what new packages were added and why. If you fix a
+   bug or add new functionality consider adding a new test to prevent the bug from
+   coming back/the feature breaking in the future.
 
-6. Once the pull request is approved (but not yet merged), Update the
+5. Once the pull request is approved (but not yet merged), Update the
    change log (details below) and commit the change log, then update
    the pull request.
```

Comment:
I've never done this changelog step, and I've never seen anyone else do it either. Should we remove it?

Reply:
+1 from me if it's not being followed

Reply:
I don't think anyone does this and we eventually gave up on having a changelog that is accurate, so let's remove it :(